.github/ISSUE_TEMPLATE/bug-issue.yml

```
@@ -0,0 +1,72 @@
+name: Bug report
+description: Report a very clearly broken issue.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+        - And most importantly, search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
+
+  - type: dropdown
+    attributes:
+      label: Type
+      options:
+        - Crash to desktop
+        - Game behaviour
+        - Performance
+        - Cosmetic
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: What are the steps for reproducing the bug? Any additional details that might be of help?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots or videos
+      description: Share any kind of visuals you can which show the bug or issue taking place.
+      placeholder: You can attach by dragging and dropping the screenshots/videos into this box.
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Version
+      description: The version you encountered this bug on. This is shown at the bottom of the main menu and also at the end of the settings screen.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Logs
+
+        Attaching log files is required for every reported bug. See instructions below on how to find them.
+
+        If the game has not yet been closed since you found the bug:
+          1. Head on to game settings and click on "Open osu! folder"
+          2. Then open the `logs` folder located there
+
+        **Logs are reset when you reopen the game.** If the game crashed or has been closed since you found the bug, retrieve the logs using the file explorer instead.
+
+        By default logs can be found in:
+          - `%AppData%/osu/logs` *(on Windows)*
+          - `~/.local/share/osu/logs` *(on Linux & macOS)*
+          - `Android/data/sh.ppy.osulazer/files/logs` *(on Android)*
```

Just to flow with the whole list better

```suggestion
        The default places to find the logs are as follows:
          - `%AppData%/osu/logs` *on Windows*
          - `~/.local/share/osu/logs` *on Linux & macOS*
          - `Android/data/sh.ppy.osulazer/files/logs` *on Android*
```